index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio :: Homepage</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/73a06aba35.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="wrapper">
+    <header>
+      <div class="time-bar"><img src="imgs/time_bar.png" alt="time-bar"></div>
+      <div class="group-bar"><img src="imgs/notification_bar.png" alt="group-bar"></div>
+      <h3 id="logo">Mansa Susa</h3>
+      <nav>
+        <i class="fa-solid fa-bars"></i>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1>Hey there.<br>I'm Susa</h1>
+      <h3>I'm a Software Engineer</h3>
+      <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <ul class="socials">
+        <li><i class="fab fa-github"></i></li>
+        <li><i class="fa-brands fa-linkedin-in"></i></li>
+        <li><i class="fa-brands fa-angellist"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li><i class="fa-brands fa-medium-m"></i></li>
```

Comment:
- [ ] Social icons look great, well done! However, it would be better to add `a` tags inside the `li` tags to make them clickable too.